.github/workflows/test.yml

```diff
@@ -228,19 +223,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macOS-latest, windows-2019, ubuntu-20.04, ubuntu-22.04]
-        ros_distribution: [foxy, humble, iron, rolling]
-        exclude:
-          - os: ubuntu-22.04
-            ros_distribution: foxy
-          - os: ubuntu-20.04
-            ros_distribution: humble
-          - os: ubuntu-20.04
-            ros_distribution: iron
-          - os: ubuntu-20.04
-            ros_distribution: rolling
+        os: [macOS-latest, windows-2019, ubuntu-22.04]
+        ros_distribution: [humble, iron, rolling]
     env:
-      DISTRO_REPOS_URL: "https://raw.githubusercontent.com/ros2/ros2/${{ matrix.ros_distribution == 'rolling' && 'master' || matrix.ros_distribution }}/ros2.repos"
+      DISTRO_REPOS_URL: "https://raw.githubusercontent.com/ros2/ros2/${{ matrix.ros_distribution }}/ros2.repos"
```

Comment:
@christophebedard Could you please explain the rationale for changes in the `DISTRO_REPOS_URL:`?  
Curios why need to remove `== 'rolling'`?